README.md

```diff
@@ -82,17 +89,30 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
-
+<sentence> ::= "the" <noun> <verb> "."
+<sentence> ::= "the" <noun> <verb> <adverb> "."
+<sentence> ::= "the" <adjective> <noun> <verb> "."
+<sentence> ::= "the" <adjective> <noun> <verb> <adverb> "."
+<noun> ::= "dog"
+<noun> ::= "cat"
+<verb> ::= "ate"
+<verb> ::= "ran"
+<adverb> ::= "slowly"
+<adverb> ::= "noisily"
+<adjective> ::= "lazy"
+<adjective> ::= "smelly"
```

Comment:
I think this is mostly correct, but it doesn't allow for repeated adjectives.

# -2

You can also save yourself some typing by using alternatives, and by pushing the decision about optional elements down the grammar:

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
  <noun> ::= "dog" | "cat"
  <verb> ::= "ate" | "ran"
  <adverb> ::= "slowly" | "noisily" | ""
~~~